stdeb/util.py

```diff
@@ -1522,6 +1534,11 @@ def build_dsc(debinfo,
 %(scripts_cleanup)s
 """
 
+RULES_OVERRIDE_PYTHON2 = """
+override_dh_python2:
+        dh_python2 --no-guessing-versions
+%(scripts)s
```

Comment:
Is there a reason you put the indent in the scripts string rather than in the format string here? When scanning this jumps out to me as wrong since the scripts should be indented in order to be part of the override_dh_python2 recipe and there's no indication on line 1105 that the scripts content is being indented for inclusion in a makefile recipe.

Reply:
Ah, I suspect it's so `scripts` doesn't leave a dangling tab when it's empty. I believe Make syntax permits such lines so I don't think it would be a problem to move the tab into the format string. But if you prefer it in the scripts variable a comment reminding future contributors why it's there is merited.

Reply:
The new code blocks simply matches the existing one in the next line for `		RULES_OVERRIDE_PYTHON3`.